x-pack/plugins/endpoint/common/types.ts

```diff
@@ -331,6 +331,26 @@ export interface EndpointEvent {
   };
 }
 
+/**
+ * Whitelist Rule
+ * https://github.com/elastic/endpoint-dev/issues/6532
```

Comment:
Now that I think about it. Aren't we not supposed to link back to private repos like this in code?

Reply:
yah. you can link it the other way. In other words, link to this PR in that issue